packages/patternfly-4/react-core/src/components/ContextSelector/ContextSelectorItem.tsx

```diff
@@ -1,40 +1,38 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import * as React from 'react';
 import styles from '@patternfly/react-styles/css/components/ContextSelector/context-selector';
 import { css } from '@patternfly/react-styles';
 import { ContextSelectorContext } from './contextSelectorConstants';
 
-const propTypes = {
+export interface ContextSelectorItemProps {
   /** Anything which can be rendered as Context Selector item */
-  children: PropTypes.node,
+  children?: React.ReactNode; 
   /** Classes applied to root element of the Context Selector item */
-  className: PropTypes.string,
+  className?: string; 
```

Comment:
I don't think we want to make things that where required previously now optional (Except for children and className)